packages/@storefront/flux-capacitor/CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] [minor]
+### Changed
+- SF-1125: Details pages now leverage the new `RequestHelper`, `productDetails`.
+
+### Added
+- SF-1125: On details pages, requests are built via `productDetails`, a new `RequestHelper` that accounts for a specified collection and area.
```

Comment:
It's not clear that the new `RequestHelper` is what was added. State the addition first, then explain.